src/Handler/TagHandler.php

```diff
@@ -57,19 +59,20 @@ public function tagResponse(Response $response, $replace = false)
         $tags = [];
         if (!$replace && $response->headers->has($this->tagsHeader)) {
             $headers = $response->headers->get($this->tagsHeader, null, false);
-            if (!empty($headers)) {
+            if ($headers) {
```

Comment:
irrelevant change?

Reply:
yes, just a micro-optimization. I think it worsts it here, as this function called on every request. 
Let me know if I should remove it.

Reply:
Then I think you should remove it.

Reason besides being irrelevant to the change here: afaik `empty` is faster, or _at least_ as fast as changing to evaluating if the value is true or false. `empty` like `isset` is language constructs, and not functions, so they don't have function overhead.

Reply:
Ok, reverted original code here and in `if (!empty($repoPrefix)) {`.

regarding `empty` vs `bool cast`, bool cast is faster (both with and without opcache optimization).
Proof:
![empty_vs_boolcast](https://user-images.githubusercontent.com/2779573/71448648-35620900-2747-11ea-9940-423e2ac649e5.png)
`!empty` variant performs 2 extra operations without opcache (just 1 extra after opcache optimization).

Reply:
I stand corrected :)